cnf-tests/testsuites/pkg/networks/network.go

```diff
@@ -0,0 +1,30 @@
+package networks
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	netattdefv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+	client "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/client"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func CreateNetworkAttachmentDefinition(cs *client.ClientSet, namespace, nadName, config string) error {
```

Comment:
consider calling the packange "networkattachementdef" and the functions Create / CreateMulti
Also, consider returning the objects and calling cs.Create from the test.